ingest.py

```diff
@@ -126,37 +126,33 @@ def process_documents(ignored_files: List[str] = []) -> List[Document]:
     print(f"Split into {len(texts)} chunks of text (max. {chunk_size} tokens each)")
     return texts
 
-def does_vectorstore_exist(persist_directory: str) -> bool:
+def ensure_integrity(persist_directory: str) -> bool:
     """
-    Checks if vectorstore exists
+    Checks if vectorstore exists, and if it does, if it is valid
     """
     if os.path.exists(os.path.join(persist_directory, 'index')):
         if os.path.exists(os.path.join(persist_directory, 'chroma-collections.parquet')) and os.path.exists(os.path.join(persist_directory, 'chroma-embeddings.parquet')):
             list_index_files = glob.glob(os.path.join(persist_directory, 'index/*.bin'))
             list_index_files += glob.glob(os.path.join(persist_directory, 'index/*.pkl'))
-            # At least 3 documents are needed in a working vectorstore
+            # More than 3 documents are needed in a working vectorstore
             if len(list_index_files) > 3:
-                return True
-    return False
+                return True # Vectorstore exists and is valid
+        return False # Vectorstore exists but is not valid
+    return True # Vectorstore does not exist yet
```

Comment:
This doesn't seem necessary, but you could add the printing inside this function.